Resources/Prototypes/_Impstation/Objectives/thief.yml

```diff
@@ -0,0 +1,453 @@
+- type: entity
+  parent: BaseThiefStealAnimalObjective
+  id: FinFinStealObjective
+  components:
+  - type: StealCondition
+    stealGroup: AnimalFinFin
+  - type: Objective
+    difficulty: 2
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: GeneratorStealObjective #tesla or singulo generator
+  components:
+  - type: NotJobRequirement
+    job: StationEngineer
+  - type: StealCondition
+    stealGroup: Generator
+    descriptionText: objective-condition-thief-generator-description
+  - type: Objective
+    difficulty: 1.5
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: PunpunIDCardStealObjective
```

Comment:
pun pun's id hardly feels like a challenge - the only difficulty is another thief rolling it. the only other unique id for thieves is the captain's which is a high value target. i realise this is 0.1 difficulty but this might as well round down to 0 and not be here